lldb/tools/lldb-dap/Handler/StackTraceRequestHandler.cpp

```diff
@@ -178,14 +179,45 @@ void StackTraceRequestHandler::operator()(
   llvm::json::Array stack_frames;
   llvm::json::Object body;
 
+  lldb::SBFormat frame_format = dap.frame_format;
+
+  if (const auto *format = arguments->getObject("format")) {
```

Comment:
Another thing I notice is the `StackFrameFormat` inherits from `ValueFormat` and thus includes the `hex?: boolean;` field as well.

I'm not sure what that would mean in terms of a stack frame however, just thought I would note that for completeness.